shell/packages/sandstorm-db/db.js

```diff
@@ -1010,20 +1010,22 @@ _.extend(SandstormDb.prototype, {
       return false;
     }
 
-    const googleDomain = this.getOrganizationGoogle();
-    const emailDomain = this.getOrganizationEmail();
+    const orgMembership = this.getOrganizationMembership();
+    const googleEnabled = orgMembership.google.enabled;
```

Comment:
It looks like `getOrganizationMembership()` could return `undefined` in which case this will throw.